src/android/org/kisio/CDVNavitiaSDKUX/CDVNavitiaSDKUX.java

```diff
@@ -30,7 +29,7 @@
     private NavitiaSDKUX navitiaSDKUX;
     private Map<String, Action> actions = new HashMap<String, Action>();
 
-    private static final String TAG = CDVNavitiaSDK.class.getName();
+    private static final String TAG = "CDVNavitiaSDK";
```

Comment:
The tag is for log purposes, it should have the classname as value. Please replace it by `private static final String TAG = CDVNavitiaSDKUX.class.getName();`